src/index.html

```diff
@@ -1,13 +1,55 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"> 
+    <link rel="preconnect" href="https://fonts.gstatic.com"> 
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+
+  <body>
+    <div class="card" data-qa="card">
+      <img
+        class="card_image"
+        src="./images/imac.jpeg"
+        alt="apple-imac"
+      >
+
+      <p class="card__tittle">
+      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="card__product-code">Код товара: 195434</p>
+
+      <div class="card__review-container">
+
+        <div class="stars card__stars">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+        </div>
+
+        <p class="card__review">Отзывов: 5</p>
+
+      </div>
+
+      <div class="card__price">
+        <p class="card__price-tittle">Цена:</p>
+        <p class="card__price-item">69 999 грн</p>
+      </div>
+
+      <a
+        class="card__button"
+        href="#"
+        data-qa="hover"
+      >Купить
+      </a>
+    </div>
+  </body>
+
 </html>
```

Comment:
there shouldn’t be empty line between child and parent element

Reply:
done